packages/manager/modules/iplb/src/frontends/iplb-frontends-edit.controller.js

```diff
@@ -247,6 +247,10 @@ export default class IpLoadBalancerFrontendsEditCtrl {
       delete request.ssl;
     }
 
+    if (request.httpHeader) {
+      request.httpHeader = request.httpHeader.split(',');
```

Comment:
no consequence to convert string in array ? It sounds weird, isn't it ?

Reply:
no, it will be used one time during the post request if something happen customer is backed to parent page with an error message. It's current implemented behavior.

No issues.